packs/ocm-klusterlet-v0.13.2/charts/ocm-klusterlet/templates/namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Namespace
```

Comment:
Palette will create the namespace by reading pack.namespace from the values.yaml. In which case, this template resource might conflict.